app/views/workflow_enhancements/_workflow_graph.html.erb

```diff
@@ -31,7 +32,7 @@
 
   <script type="text/javascript">
   $(function() {
-    var json = <%= WorkflowEnhancements::Graph.load_data(roles, trackers, issue).to_json.html_safe %>;
+    var json = <%= WorkflowEnhancements::Graph.load_data(roles, trackers, issue, project_roles).to_json.html_safe %>;
```

Comment:
Don't calculate project_roles in the view, just pass the `@project` object and do things in the Graph.